cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/stats/BrokerStats.java

```diff
@@ -122,17 +127,18 @@ public void discardIrrelevantResponse(CruiseControlParameters parameters) {
   @Override
   public String toString() {
     StringBuilder sb = new StringBuilder();
-    boolean hasDiskInfo = !_brokerStats.get(0).capacityByDisk().isEmpty();
+    boolean hasDiskInfo = !_brokerStats.get(0).diskStatsByLogdir().isEmpty();
 
     // put broker stats.
-    sb.append(String.format("%n%n%" + _hostFieldLength + "s%15s%26s%15s%25s%25s%20s%20s%20s%40s%n",
-                            "HOST", "BROKER", "DISK(MB)/_(%)_", "CPU(%)", "LEADER_NW_IN(KB/s)",
-                            "FOLLOWER_NW_IN(KB/s)", "NW_OUT(KB/s)", "PNW_OUT(KB/s)", "LEADERS/REPLICAS",
-                            hasDiskInfo ? "LOGDIR -> DISK(MB)/_(%)_" : ""));
+    sb.append(String.format("%n%n%" + _hostFieldLength + "s%15s%" + _logdirFieldLength + "s%26s%15s%25s%25s%20s%20s%20s%n",
+                            "HOST", "BROKER", hasDiskInfo ? "LOGDIR" : "", "DISK(MB)/_(%)_", "CPU(%)", "LEADER_NW_IN(KB/s)",
+                            "FOLLOWER_NW_IN(KB/s)", "NW_OUT(KB/s)", "PNW_OUT(KB/s)", "LEADERS/REPLICAS"));
     for (SingleBrokerStats stats : _brokerStats) {
-      sb.append(String.format("%" + _hostFieldLength + "s,%14d,%19.3f/%05.2f,%14.3f,%24.3f,%24.3f,%19.3f,%19.3f,%14d/%d%n",
+      sb.append(String.format("%" + _hostFieldLength + "s,%14d,%" + _logdirFieldLength
+                              + "s%19.3f/%05.2f,%14.3f,%24.3f,%24.3f,%19.3f,%19.3f,%14d/%d%n",
                               stats.host(),
                               stats.id(),
+                              "",
```

Comment:
Is it intentionally always empty?

Reply:
Yes, for each broker, we print broker statistics in the first line.
Then if logdir information is populated, we print disk statistics one-per-line.